test/provision/k8s_install.sh

```diff
@@ -94,8 +94,11 @@ deb http://apt.kubernetes.io/ kubernetes-xenial main
 EOF
 
 sudo rm /var/lib/apt/lists/lock || true
-retry_function "wget https://packages.cloud.google.com/apt/doc/apt-key.gpg"
-apt-key add apt-key.gpg
+# retry_function "wget https://packages.cloud.google.com/apt/doc/apt-key.gpg"
+# apt-key add apt-key.gpg
+# the Google Cloud SDK apt-key is not working at the moment (error 500)
+# temporarily hardcode the latest version of the key available (taken from my personal machine, so we're OK @Nicolas)
```

Comment:
@nbusseneau can we hardcode 
```suggestion
# temporarily hardcode the latest version of the key available (taken from https://web.archive.org/web/20230223152417/https://packages.cloud.google.com/apt/doc/apt-key.gpg)
```

Reply:
@nbusseneau but this should still be changed otherwise if we ever need to change this for some reason we will have to call you to get a new key from your personal machine.

Reply:
This is a kludge for until Google Cloud SDK fixes their stuff at which point we should revert the commit. If we ever need to update that base64-encoded key it means something went very, very wrong 😁